src/integration/pages/onDemandRadioPage/commonRecentEpisodesTests.js

```diff
@@ -0,0 +1,23 @@
+export default () => {
+  describe('Recent Episodes', () => {
+    const recentEpisodesList = document.querySelector(
+      '[data-e2e="recent-episodes-list"]',
+    );
+
+    const recentEpisodesListItems = document.querySelectorAll(
+      '[data-e2e="recent-episodes-list-item"]',
+    );
+
+    it('should be in the document', () => {
+      expect(recentEpisodesList).toBeInTheDocument();
+    });
+
+    it('should contain the list items', () => {
+      recentEpisodesListItems.forEach(itemEl => {
+        expect(itemEl).toBeInTheDocument();
+        expect(itemEl.textContent).toBeTruthy();
```

Comment:
I'd also snapshot textContent as it's maybe not enough of a test top check it's truthy